Jenkinsfiles/web-ui-pipeline.groovy

```diff
@@ -0,0 +1,210 @@
+properties([
+    [$class: 'RebuildSettings', autoRebuild: false, rebuildDisabled: false],
+    parameters([
+            string(name: 'BRANCH', defaultValue: '', description: 'Branch to test, fall-backs on $BASE_BRANCH if not found.', trim: false),
+            choice(name: 'BASE_BRANCH', choices: ['9.10'], description: 'The branch to fallback on when $BRANCH is not found.'),
+            string(name: 'SLAVE', defaultValue: 'SLAVE910', description: 'Slave label to be used.', trim: false),
+            booleanParam(name: 'CLEAN', defaultValue: false, description: 'Run npm and bower cache clean?'),
+            booleanParam(name: 'SAUCE_LAB', defaultValue: true, description: 'Should unit tests be run on Sauce Lab (or just Chrome on the slave)?'),
+            booleanParam(name: 'CREATE_PR', defaultValue: true, description: 'Should PRs be created if build is successful?'),
+            booleanParam(name: 'SKIP_IT_TESTS', defaultValue: false, description: 'Should the functional tests be skipped?'),
+            booleanParam(name: 'SKIP_UNIT_TESTS', defaultValue: false, description: 'Should the element\'s tests be skipped?'),
+            booleanParam(name: 'GENERATE_METRICS', defaultValue: false, description: 'Should the metrics report be generated?'),
+            string(name: 'BROWSER', defaultValue: 'chrome', description: 'The browser to use for functional tests.', trim: false),
+            string(name: 'BROWSER_BINARY', defaultValue: '', description: 'The path to the browser binary.', trim: false),
```

Comment:
Sama same? @Gabez0r ?

Reply:
[This one is available too](https://github.com/nuxeo/plugin-nuxeo-web-ui/blob/2.2_9.10/nuxeo-web-ui-ftest/wdio.conf.js#L65).